gradle/sonar.gradle

```diff
@@ -126,6 +126,20 @@ project(":apiml-common-lib-package") {
     }
 }
 
+// Packaging project; sonar does not apply
```

Comment:
I am not sure if we want to include this project in Sonar. I probably wouldn't do that

Reply:
yeah, I was thinking so, that's why it is excluded